File: JavaScript-Node/uri-1007.js
// URI - 1007

const readline = require('readline')

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
})

rl.on('line', input => {
    const aValue = parseInt(input)

    rl.on('line', input => {
        const bValue = parseInt(input)

        rl.on('line', input => {
            const cValue = parseInt(input)

            rl.on('line', input => {
                const dValue = parseInt(input)
                const difference = parseInt((aValue * bValue - cValue * dValue))

                console.log('DIFERENCA = ' + difference)

                rl.close()
            })
        })
    })
})

File: URI/C#/uri-1007.cs
// URI - 1007

using System;

namespace URI
{
    class Program
    {
        static void Main(string[] args)
        {
            int aValue = int.Parse(Console.ReadLine());
            int bValue = int.Parse(Console.ReadLine());
            int cValue = int.Parse(Console.ReadLine());
            int dValue = int.Parse(Console.ReadLine());

            Console.WriteLine("DIFERENCA = " + (aValue * bValue - cValue * dValue));
        }
    }
}